command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
```

Comment:
We should discuss whether `upload` and `download` should be part of `did`. They are the most complex commands in the client, so it wouldn’t be unwarranted to separate them. They are verbs, so there is a clear distinction with the groups. Plus, an upload not only adds DIDs, but also replicas, replication rules, and maybe other entities as well.